### ISSUE TITLE
NoSpawn Tag Optional

### ISSUE THREAD

**Issue by TinyPandas** [vadis365/Mob-Grinding-Utils#204]
Would it be viable to implement so enabled/disabled element for this tag on the Delightful/Dreadful Dirt(s) and Spawner?

In reference to [this](https://github.com/vadis365/Mob-Grinding-Utils/commit/44b48336195112eab32cc352116caf6030dd596f)

**Comment by Flanks255** [comment]
Could you clarify what you are talking about? the tags have no bearing on anything outside of the spawner, the swap, and dreadful dirt, and by default, is empty.

**Comment by Flanks255** [comment]
bees spawning from dirt? or in general from hives?

**Comment by tctfox** [comment]
> bees spawning from dirt? or in general from hives?

Bee's spawn normally. But they do not spawn from delightful dirt

**Comment by Flanks255** [comment]
ya, there is no way the addition of these tags would prevent them from spawning on it, unless some mod, or datapack, is adding them to the tag.

**Comment by tctfox** [comment]
> Could you clarify what you are talking about? the tags have no bearing on anything outside of the spawner, the swap, and dreadful dirt, and by default, is empty.

I dont know what exactly changed. But with that update the spawning from delightful dirt broke for some mobs. Mobs such as bees (vanilla and modded) do not spawn anymore. I think it's because of them not spawning the nornal way but through hives. 

I am not sure if it is the tag or something else in the update but I don't see anything else on the changes 

> ya, there is no way the addition of these tags would prevent them from spawning on it, unless some mod, or datapack, is adding them to the tag.

Hm alright interesting. I am playing AllTheMods6 and reverting your mod version to a version prior to march it fixes it. 

Thanks for your help tho

**Comment by tctfox** [comment]
> The reason i see why they arent spawning, is we check for MobCategory.CREATURE, which is passive mobs, and resourceful bees makes its own bee category. but that hasnt changed, its always been that way.

Yeah I don't get it either. I just tried again and on the newest it doesn't work and on an older it works. Only changing the MGU version. 

But I can just play on the older version thats fine by me
